src/devcenter/setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '1.0.2'
+VERSION = '1.0.3'
```

Comment:
It's a breaking change to rename argument option name. You should use version `2.0.0` instead

Reply:
Updated